app/views/coronavirus_local_restrictions/_tier_one_restrictions.html.erb

```diff
@@ -17,14 +17,14 @@
 
 <%= tag.div :data => {
   :module => "coronavirus-track-local-restriction-results",
-  "tracking-label" => "Tier one: #{location_lookup.lower_tier_area_name}"
+  "tracking-label" => "Tier one: #{restriction.area_name}"
```

Comment:
Is the `lower_tier_area_name` method in the LookupService still needed? Should we get rid of it?